### ISSUE TITLE
The CseClientHttpRequest automatically adds the path variable named ‘null’.

### ISSUE THREAD

**Issue by liangrongjie** [apache/servicecomb-java-chassis#1215]
When the Cse is used as the client to send a request without the path variable to other microservices, the cse automatically adds a redundant variable named "null".

The original path`/v1/countries` is changed to `/v1/countries?null`.

The code is in the `org.apache.servicecomb.provider.springmvc.reference.CseClientHttpRequest#prepareInvocation(Object[])`
```java
invocation.getHandlerContext().put(RestConst.REST_CLIENT_REQUEST_PATH,
        path + "?" + this.uri.getRawQuery());
```

when request path  without the path variable, the ` this.uri.getRawQuery()` return null (*not string ""*)；executes the request at the `org.apache.servicecomb.transport.rest.client.http.RestClientInvocation#createRequestPath(RestOperationMeta)`   obtained incorrect path from invocation.handlerContext.

```java
String path = (String) invocation.getHandlerContext().get(RestConst.REST_CLIENT_REQUEST_PATH);
```

**Comment by liangrongjie** [comment]
https://issues.apache.org/jira/browse/SCB-1289

**Comment by yhs0092** [comment]
An Apache jira issue is created to track it [SCB-1289](https://issues.apache.org/jira/browse/SCB-1289)
Could you please provide a PR to fix it?

**Comment by liubao68** [comment]
This issue has bean resolved in 1.3.0. Please testing using this version and feel free to create an new issue if something goes wrong.
